src/checkUrlParams.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable max-len */
+const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function checkUrlParams(textToConvert, toCase) {
+  const errors = [];
+
+  if (!textToConvert) {
+    errors.push({ message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".' });
```

Comment:
You can create an object with all your default error messages and use them as reference